CONTRIBUTING.md

```diff
@@ -180,6 +180,48 @@ See [Translating](#translating) for more details.
 
 There is a dedicated [`apache-superset` tag](https://stackoverflow.com/questions/tagged/apache-superset) on [StackOverflow](https://stackoverflow.com/). Please use it when asking questions.
 
+## Types of Contributors
+
+Following Apache Software Foundation's project governance, Apache Superset has the following types of contributors
+
+### PMC Member
+
+A Project Management Committee (PMC) member is a person who has been elected by the PMC to help manage the project. PMC members are responsible for the overall health of the project, including community development, release management, and project governance. PMC members are also responsible for the technical direction of the project.
+
+For more details on PMC members, please refer to https://www.apache.org/foundation/governance/pmcs.html
+
+### Committer
+
+A committer is a person who has been elected by the PMC to have write access (commit access) to the code repository. they can modify the code, documentation, and website by themselves and also accept other contributions.
+
+The official list of committers and PMCs can be found [here](https://projects.apache.org/committee.html?superset).
```

Comment:
```suggestion
The official list of committers and PMC members can be found [here](https://projects.apache.org/committee.html?superset).
```

Reply:
> You can also see the Superset project's Committer nomination guidelines here:
https://github.com/apache/superset/wiki/Standards-for-evaluating-a-potential-Committer-to-the-Apache-Superset-project